apps/dhparam.c

```diff
@@ -313,29 +313,29 @@ int dhparam_main(int argc, char **argv)
                         "# include <openssl/dh.h>\n"
                         "#endif\n"
                         "\n");
-        BIO_printf(out, "DH *get_dh%d()\n{\n", bits);
+        BIO_printf(out, "static DH *get_dh%d()\n{\n", bits);
         print_bignum_var(out, pbn, "dhp", bits, data);
         print_bignum_var(out, gbn, "dhg", bits, data);
         BIO_printf(out, "    DH *dh = DH_new();\n"
-                        "    BIGNUM *dhp_bn, *dhg_bn;\n"
+                        "    BIGNUM *p, *g;\n"
```

Comment:
Is the variable rename really necessary?

Reply:
Considering that dh is normally described with `p` and `g` (and `q`), I think the rename is appropriate.

Reply:
I'm fine with this.  I didn't think the _dh_ or the _bn_ suffix interfered with interpretation.